generators/entity-client/templates/angular/src/main/webapp/app/entities/entity-management-update.component.ts.ejs

```diff
@@ -216,8 +216,8 @@ _%>
     setFileData(event, field: string, isImage) {
         return new Promise((resolve, reject) => {
             if (event && event.target && event.target.files && event.target.files[0]) {
-                const file = event.target.files[0];
-                if (isImage && !/^image\//.test(file.type)) {
+                const file: File = event.target.files[0];
+                if (isImage && !file.type.startsWith('image/')) {
```

Comment:
why did you change it?

Reply:
It reports an error due to typescript eslint [rule](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/prefer-string-starts-ends-with.md) that's enabled by default as recommended configuration. I also prefer `startsWith` over regex in such scenarios as it's more readable.